client/lib/plans/constants.js

```diff
@@ -1279,7 +1279,7 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_BLOG_DOMAIN ]: {
 		getSlug: () => FEATURE_BLOG_DOMAIN,
-		getTitle: () => i18n.translate( 'Custom .blog Domain' ),
+		getTitle: () => i18n.translate( 'Free .blog Domain for One Year' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Free .blog domain for one year' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Capitalization is important here.